content/en/tutorial/05-refs.md

```diff
@@ -177,7 +177,7 @@ function App() {
 
   return (
     <div>
-      <input initialValue="Hello World!" />
+      <input value="Hello World!" />
```

Comment:
```suggestion
      <input defaultValue="Hello World!" />
```

No real difference in this example, but I believe `defaultValue` is what they were mentally reaching for.

Reply:
Thanks. My first guess was defaultValue (and it appeared to work), but then I checked https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attributes and didn't see it there, so I thought that might be some kind of unofficial behavior. But I see it's listed on https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement, so I guess the omission there was a mistake.

Reply:
`defaultValue` is a property, rather than an attribute. The distinction is the interface, as attributes are an HTML thing while properties are JS(X). It therefore gives access to a few things in an HTML-like interface that wouldn't actually work in HTML.